mx-rust-semantics/main/expression/rust-to-mx.md

```diff
@@ -0,0 +1,37 @@
+```k
+
+module MX-RUST-EXPRESSION-RUST-TO-MX
+    imports private COMMON-K-CELL
+    imports private K-EQUAL-SYNTAX
+    imports private LIST
+    imports private MAP
+    imports private MX-COMMON-SYNTAX
+    imports private MX-RUST-REPRESENTATION-CONVERSIONS
+    imports private RUST-CONVERSIONS-SYNTAX
+    imports private RUST-EXECUTION-CONFIGURATION
+    imports private RUST-REPRESENTATION
+
+    rule V:MxValue ~> rustToMx => rustToMx(V)
+
+    rule
+        <k>
+            rustToMx(struct (_, Fields:Map))
+            => rustValuesToMxListValue
+                ( ptrListToValueList(listToPtrList(values(Fields)), Values)
+                , .MxValueList
+                )
+            ...
+        </k>
+        <values> Values:Map </values>
+        [priority(200)]
```

Comment:
Minor nitpick, is there a reason to use `priority` instead of `owise` here? Having `priority(200)` is equivalent to just using `owise` ([reference](https://kframework.org/k-distribution/k-tutorial/1_basic/07_side_conditions/#rule-priority)).

Reply:
fixed